### PR TITLE
Unstructured Sedna Lifelong Learning Architecture

### PR DESCRIPTION
**Pull request by luosiqi**
In this PR, unstructured Sedna lifelong learning architecture is achieved.  It surpports the following new features.
* Unstructured lifelong learning
* Unseen task detection
* Online unseen task processing
* Offline unseen task training

### CODE REVIEW

File: examples/lifelong_learning/robot_dog_delivery/RFNet/basemodel.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from utils.args import TrainingArguments, EvaluationArguments
 from estimators.train import Trainer
 from estimators.eval import Validator, load_my_state_dict
-from accuracy import robo_accuracy
+from accuracy import accuracy
 
 os.environ["BACKEND_TYPE"] = ''
```
Comment (JimmyYang20) [inline examples/lifelong_learning/robot_dog_delivery/RFNet/basemodel.py:22]: env var's default value is ""

File: lib/sedna/algorithms/seen_task_learning/task_allocation/base_task_allocation.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""
+Mining tasks of inference sample base on task attribute extractor
```
Comment (MooreZheng) [inline lib/sedna/algorithms/seen_task_learning/task_allocation/base_task_allocation.py:2]: base on -> based on

File: examples/lifelong_learning/robot_dog_delivery/README.md [hunk 3]
```diff
@@ -1,201 +1,226 @@
...
-
-These images are generated by the script [build_images.sh](/examples/build_image.sh).
-
-### Create Lifelong Learning Job
-
-#### Preparations
-Before this, user must provide an s3 directory for storage which is denoted as "s3_prefix" in this example, by setting environment variable s3_prefix.
-```
-s3_prefix=$s3_prefix
-cloud_image=swr.cn-south-1.myhuaweicloud.com/sedna/sedna-robo:v0.1.1
```
Comment (MooreZheng) [inline examples/lifelong_learning/robot_dog_delivery/README.md:38]: Add an introduction to the images (as introduced below

File: lib/sedna/algorithms/seen_task_learning/task_allocation/base_task_allocation.py (hunk 2 above)
Comment (luosiqi) [reply lib/sedna/algorithms/seen_task_learning/task_allocation/base_task_allocation.py:2]: done

File: examples/lifelong_learning/robot_dog_delivery/README.md (hunk 3 above)
Comment (luosiqi) [reply examples/lifelong_learning/robot_dog_delivery/README.md:38]: done

File: examples/lifelong_learning/robot_dog_delivery/RFNet/basemodel.py (hunk 1 above)
Comment (luosiqi) [reply examples/lifelong_learning/robot_dog_delivery/RFNet/basemodel.py:22]: done

**Comment by MooreZheng** [comment]
My comment is addressed in the current version. 
/lgtm

**Comment by jaypume** [comment]
For commit history of Sedna repository more readable and cleaner, I suggest to squash the commits by git rebase -i kubeedge/main, to keep the number of commit to 3~5, and to make the commit message more concrete, instead of "update this... imporve that"

**Pull request merged**